test/box/access.test.lua

```diff
@@ -841,6 +841,10 @@ _ = box.schema.func.call("LUA", "return box.space._space:count()")
 _ = box.schema.func.call("box.schema.user.info", 0)
 _ = box.schema.func.call("box.schema.user.info", 1)
 session.su('admin')
+c = net.connect(LISTEN.host, LISTEN.service)
+c:call('LUA', {'box.session.su("admin") return 1'})
```

Comment:
I suggest to use this instead:
```
c:call('LUA', {'return 1'})
```

I mean the result is the same, but the current line looks more defiant.

Reply:
OK. I just wanted to ensure that a user can't get admin rights.